src/multi_vector_simulator/F2_autoreport.py

```diff
@@ -715,7 +708,6 @@ def create_app(results_json, path_sim_output=None):
                         className="cell imp_info",
                         children=[
                             html.P(f"MVS Release: {version_num} ({version_date})"),
-                            html.P(f"Branch-id: {branchID}"),
```

Comment:
But printing the used branch in the report was important for me, so that we do not mess up comparing two scenarios that are used with different dev versions or something. Is removing it necessary?

Reply:
If you run the simulation after installing the package with pip, you might run a simulation not from the root of this repository, therefore the code to fetch the branch id of this repository fails because you are not anymore within a repository when you run the code.

Ultimately, the version number is the only thing which matters (it is uniquely mapped to a commit id in the `master` branch of the repo) as on pip you cannot have two release with same version numbers, you can only bump version number up.

If you want to keep the branch-id feature, we should pack this into a function of utils which detects if the person executing the code does it from the root of the repository or not and provide, or not, the branch id

Reply:
Mhm... okay. Actually, as the end-user will use the simplest way to install the MVS, we probably do not need to print the branch-id. 

For devs: When working on the dev branch, we have the same version number as the master release, dont we? Could we not bump the version number of dev to something like `vX.Y.Z.dev`? and then we are at least warned when we compare dev to real releases. Additionally, we can then check the simulation date to guess if large changes might already have been applied to dev...

Reply:
We can reintroduce the ID in a subsequent PR what do you say?

Reply:
You mean as in "introduce in another way"? Sounds fine. Can you create an issue noting down your thoughts?

Otherwise it would be better to just revieve it now.